_includes/header.html

```diff
@@ -61,334 +61,23 @@
-                                            <li><a tabindex="-1" href="newsletter/shop/newsletter-shop-2.html" target="_blank">Newsletter Shop: Purple</a></li>
-                                            <li><a tabindex="-1" href="newsletter/shop/newsletter-shop-3.html" target="_blank">Newsletter Shop: Blue</a></li>
-                                            <li><a tabindex="-1" href="newsletter/shop/newsletter-shop-4.html" target="_blank">Newsletter Shop: Green</a></li>
-                                            <li><a tabindex="-1" href="newsletter/shop/newsletter-shop-5.html" target="_blank">Newsletter Shop: Yellow</a></li>
-                                            <li><a tabindex="-1" href="newsletter/shop/newsletter-shop-6.html" target="_blank">Newsletter Shop: Orange</a></li>
-                                        </ul>
-                                    </li>
-                                </ul>
-                            </li>
-                            <li class="dropdown-submenu">
-                                <a tabindex="-1" href="#">Components</a>
-                                <ul class="dropdown-menu">
-                                    <li><a tabindex="-1" href="wp-accordion.html">Accordion</a></li>
-                                    <li><a tabindex="-1" href="wp-alerts.html">Alerts</a></li>
-                                    <li><a tabindex="-1" href="wp-badges.html">Badges</a></li>
-                                    <li><a tabindex="-1" href="wp-blocks-default.html">Blocks</a></li>
-                                    <li><a tabindex="-1" href="wp-buttons-boomerang.html">Buttons</a></li>
-                                    <li><a tabindex="-1" href="wp-callouts.html">Callouts</a></li>
-                                    <li><a tabindex="-1" href="wp-carousels.html">Carousels</a></li>
-                                    <li><a tabindex="-1" href="wp-counters.html">Counters</a></li>
-                                    <li><a tabindex="-1" href="wp-pagination.html">Pagination</a></li>
-                                    <li><a tabindex="-1" href="wp-progress-bars.html">Progress bars</a></li>
-                                    <li><a tabindex="-1" href="wp-range-sliders.html">Range sliders</a></li>
-                                    <li><a tabindex="-1" href="wp-rating-stars.html">Rating stars</a></li>
-                                    <li><a tabindex="-1" href="wp-ribbons.html">Ribbons</a></li>
-                                    <li><a tabindex="-1" href="wp-tabs.html">Tabs</a></li>
-                                </ul>
-                            </li>
-                            <li><a href="wp-typography-general.html">Typography</a></li>
-                            <li><a href="wp-features.html">Features</a></li>
-                        </ul>
+                    <li>
+                        <a href="/services">Services</a>
```

Comment:
This should have the pop out drop down menu like https://github.com/secseattle/secseattle.github.io/pull/17/files#diff-822e9cf54ee310f9a7ebd0d16c9ae066L70 the `mega-nav-section` using the 3 images.